Libraries/pbxsetting/Sources/DefaultSettings.cpp

```diff
@@ -14,7 +14,9 @@
 
 #include <cstdlib>
 
+#if defined(__APPLE__)
```

Comment:
It's just for `confstr()` which is Darwin-only. Everything should go through the `process::Context`, but that isn't exposed there so the `confstr()` is accessed directly.